sections/03_categorize.md

```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
 Additionally, unique barriers exist in this space that may hinder progress in
 this field.
 
+###### Lack of ground-truth labels in data, and high cost in post-study validation by clinician experts.
+
+*Lack of true labels is perhaps among the biggest obstacles for electronic 
+health records(EHR)-based learing studies (e.g., phenotyping). Popular deep 
+learning (and machine learning) methods are mostly classification problem, 
+which require ground-truth labels (i.e., response variables) for training. 
+For EHR, unfortunately, this means we need to hire multiple expert clinicians 
+to manually read and annotate individual patients' records to assign "true" 
+labels (i.e., expert chart review). Sometimes, the learnt features also need to 
+be manually validated and interpreted by clinicians. This can be quite costly 
```

Comment:
@traversc - I think you can file a pull request against @XieConnect's repository to mention #170. As soon as that gets approved, I think your contribution would show up here.